src/utils/messages.js

```diff
@@ -292,6 +292,12 @@ export function cannotInstallWorkspaceInProject(pkgName: string): Message {
   return `Cannot install workspace "${pkgName}" as a dependency of a project`;
 }
 
+export function cannotUpgradeWorkspaceDependencyInProject(
+  pkgName: string
+): Message {
+  return `Cannot upgrade workspace "${pkgName}" as a dependency of a project`;
```

Comment:
Could we add a tiny bit more info here?

"Cannot upgrade workspace "${pkgName}" as a dependency of a project, workspaces are all symlinked, upgrading would not make sense"

^^ Not the best message either, but I think its better than throwing terms the user might now know.